### PR TITLE
Maintenance banner for website outage

### PR DESCRIPTION
**Pull request by mauzey1**
This is a banner for announcing the outage of the site for server maintenance on October 4-6, 2019.

### CODE REVIEW

**Comment by mauzey1** [comment]
I will come up with a site-wide banner rather than a message at the front page.  I will close this PR now.

**Comment by durack1** [comment]
@mauzey1 thanks for doing this - if you have a screenshot to drop into the PR that will be a more useful review than the blind reviews I normally complete

File: _includes/maintenance.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="hero-unit">
+  <p>
+    Due to a scheduled maintenance, complete power outage will occur the weekend of Oct. 4-6, 2019 . As a result, the webserver will be going down around noon Friday 10/4. The webserver will be brought back up Monday morning, Oct. 7.
```
Comment (durack1) [inline _includes/maintenance.html]: @mauzey1 I would amend to read"
```
Due to scheduled maintenance, a website outage will occur from noon Friday 4th October 2019 through early am Monday 7th October 2019.

Apologies in advance for any inconvenience
```

**Comment by mauzey1** [comment]
@durack1 Here's a banner idea
Screenshot-2019-9-13 PCMDI - Metrics

**Comment by durack1** [comment]
@mauzey1 yep, looks good, I just approved #316

**Pull request closed without merging**